system/Validation/Validation.php

```diff
@@ -343,7 +343,50 @@ public function hasRule(string $field): bool
 
 	//--------------------------------------------------------------------
 
-    /**
+	/**
+	 * Get rule group.
+	 *
+	 * @param string $group Group.
+	 *
+	 * @return string[] Rule group.
+	 *
+	 * @throws \InvalidArgumentException If group not found.
+	 */
+	public function getRuleGroup(string $group): array
+	{
+		if (!isset($this->config->$group)) {
```

Comment:
It's a small, pedantic thing, but the style says that `!` should be followed by a space. Additionally, the opening curly braces on functions should be on it's own line.